src/Features/CSharp/Portable/ExtractMethod/CSharpSelectionResult.ExpressionResult.cs

```diff
@@ -25,9 +25,8 @@ internal abstract partial class CSharpSelectionResult
         private sealed class ExpressionResult(
             SemanticDocument document,
             SelectionType selectionType,
-            TextSpan finalSpan,
-            bool selectionChanged) : CSharpSelectionResult(
```

Comment:
remove this valiue.  it was there to just signal data to tests, but iddn't affect behavior in any way.